source/bson-dbref/dbref.md

```diff
@@ -1,4 +1,4 @@
-# Handling of DBRefs
+# BSON DBRef
```

Comment:
Nit: Looking at other "BSON \<type\>" specs, this suggests that `DBRef` is a BSON type, which it isn't. Any objections to just calling this the "DBRef spec"?

Reply:
Ah, right, I was confusing it with the deprecated `DBPointer`.